plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -102,9 +108,14 @@ sentry_value_t HandleBeforeCrash(const sentry_ucontext_t *uctx, sentry_value_t e
 		USentryEvent* EventToProcess = NewObject<USentryEvent>();
 		EventToProcess->InitWithNativeImpl(eventDesktop);
 
-		return SentrySubsystem->GetBeforeSendHandler()->HandleBeforeSend(EventToProcess, nullptr)
-			? event
-			: sentry_value_new_null();
+		USentryEvent* ProcessedEvent = EventToProcess;
+		if(!FUObjectThreadContext::Get().IsRoutingPostLoad)
+		{
+			// Executing UFUNCTION is allowed only when not post-loading
```

Comment:
Could you provide some context what this means?

Reply:
Post-loading is a part of an actor lifecycle that goes right after finishing loading it from disk (i.e. when a game map opened in editor). Typically at this point some related actor's versioning or fixup behavior should be called. It's unsafe to use any blueprint-implemented functions (like `HandleBeforeSend` in our case) during this stage so we've added an extra check here to avoid triggering a corresponding assertion.

Reply:
Thanks for the explanation!